pandas/core/indexing.py

```diff
@@ -86,7 +86,8 @@ class IndexingError(Exception):
 
 
 class IndexingMixin:
-    """Mixin for adding .loc/.iloc/.at/.iat to Datafames and Series.
+    """
+    Mixin for adding .loc/.iloc/.at/.iat to Datafames and Series.
     """
```

Comment:
do we have a policy on whether this should be one line or not?

Reply:
there is `D200 One-line docstring should fit on one line with quotes`, see http://www.pydocstyle.org/en/5.0.2/error_codes.html, but I think that would go against the code cleanups in #31162, #31462 and others.

we want consistency. so we need a policy.

Reply:
there are ~550 one-line docstrings split over 3 lines.